prime-router/src/main/kotlin/fhirengine/translation/hl7/utils/ConstantResolver.kt

```diff
@@ -131,8 +160,13 @@ class FhirPathCustomResolver : FHIRPathEngine.IEvaluationContext {
         throw NotImplementedError("Not implemented")
     }
 
-    override fun resolveFunction(functionName: String?): FHIRPathEngine.IEvaluationContext.FunctionDetails {
-        throw NotImplementedError("Not implemented")
+    override fun resolveFunction(functionName: String?): FunctionDetails? {
+        check(!functionName.isNullOrBlank())
+        return when (functionName) {
+            "RSAddressUse" -> FunctionDetails("convert FHIR address type to HL7", 0, 0)
```

Comment:
Do not call these RS functions.  This library will be later moved to its own repo and be on its own separate from RS.  Maybe "getAddressUse"?